src/css/application.css

```diff
@@ -68,6 +68,9 @@
   --color-yellow: #ffd100;
   --color-red: #ff5e4f;
   --color-purple: #f0f;
+  --color-low-contrast-blue: #baecff;
+  --color-blue: #00b8ff;
+  --rounded-corners: 0.2em;
```

Comment:
How about `--size-rounded-corners` to maintain the naming convention established by `--color-` prefixed variables above?